packages/edit-site/src/components/dataviews/view-actions.js

```diff
@@ -176,6 +176,63 @@ function FieldsVisibilityMenu( { view, onChangeView, fields } ) {
 	);
 }
 
+function FiltersVisibilityMenu( { view, onChangeView, fields } ) {
+	const filtersHidable = fields
+		.filter(
+			( f ) =>
+				f.filters &&
+				f.filters.length > 0 &&
+				( ! f.hasOwnProperty( 'enableFilterHiding' ) ||
```

Comment:
I'd expect this to be the default behavior, so maybe inverse the property to something like `disableFilterHiding`?